doc/ChangeLog

```diff
@@ -1,4 +1,71 @@
 ===============================================================
+Tag name: ctsm5.3.078
+Originator(s): slevis (Samuel Levis,UCAR/TSS,303-665-1310)
+Date: Thu 02 Oct 2025 05:19:57 PM MDT
+One-line Summary: Merge b4b-dev to master
+
+Purpose and description of changes
+----------------------------------
+
+ PR #3516 by Matvey: Add failing test as reminder for mosart history issue
+ PR #3511 by Erik: Decompinit timer updates
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm6_0
+
+[ ] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[ ] clm4_5
+
+
+Bugs fixed
+----------
+List of CTSM issues fixed (include CTSM Issue # and description) [one per line]:
+Fixes #3494 ERI_Ld90 tests fail with mosart.h0a having diffs
```

Comment:
```suggestion
```
That PR does not fix the issue, It just makes it visible during testing.

Reply:
Right, good point!

Reply:
Thank you @mvdebolskiy!